packages/admin/admin/src/common/toolbar/Toolbar.tsx

```diff
@@ -56,18 +83,34 @@ const MainContentContainer = createComponentSlot("div")<ToolbarClassKey>({
 `);
 
 export const Toolbar = (inProps: ToolbarProps) => {
-    const { children, elevation = 1, slotProps, ...restProps } = useThemeProps({ props: inProps, name: "CometAdminToolbar" });
+    const {
+        children,
+        hideTopBar = false,
+        hideBottomBar: passedHideBottomBar,
+        elevation = 1,
+        slotProps,
+        scopeIndicator,
+        ...restProps
+    } = useThemeProps({ props: inProps, name: "CometAdminToolbar" });
     const { headerHeight } = React.useContext(MasterLayoutContext);
+    const hideBottomBar = passedHideBottomBar ?? React.Children.count(children) === 0 ?? false;
 
     const ownerState: OwnerState = {
         headerHeight,
     };
 
     return (
         <Root elevation={elevation} ownerState={ownerState} {...slotProps?.root} {...restProps}>
-            <StyledToolbar {...slotProps?.muiToolbar}>
-                <MainContentContainer {...slotProps?.mainContentContainer}>{children}</MainContentContainer>
-            </StyledToolbar>
+            {!hideTopBar && (
+                <TopBar>
+                    <ToolbarBreadcrumbs scopeIndicator={scopeIndicator} />
```

Comment:
Maybe it makes sense to separate the scope-indicator from the breadcrumbs...
Should be possible, if we pass a `ref` of the `Root` slot into the breadcrumbs.

We could try this in a separate PR.

Reply:
Okay, let's do that in a follow-up PR